plugins/kontent-plugin/README.md

```diff
@@ -0,0 +1,309 @@
+<a href="https://www.gatsbyjs.com">
+  <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
+</a>
+
+# Gatsby Starter Kontent Homepage
+
+Create a homepage using Gatsby and Kontent. This starter demonstrates how to use Kontent to build a homepage and can be customized to match your own visual branding.
+
+[View the Demo](https://gatsbykontenthomepage.gatsbyjs.io/)
+
+**Note:**
+This version of the Kontent homepage starter is written in JavaScript. If you want to use Kontent but TypeScript is more your style, there is also a TypeScript version maintained on [GitHub](https://github.com/gatsbyjs/gatsby-starter-kontent-homepage-ts).
+
+## Quick start
+
+To use this starter, you will need a new or existing [Kontent.ai project][] and then you will be asked for your [project id][].
+
+[kontent.ai project]: https://kontent.ai/learn/tutorials/manage-kontent/projects/manage-projects/#a-create-projects
+[project id]: https://kontent.ai/learn/tutorials/develop-apps/get-content/get-content-items/#a-1-find-your-project-id
+
+1. **Create a Gatsby site**
+
+   Use the Gatsby CLI to get started locally:
+
+   ```sh repo
+   npx gatsby new my-homepage https://github.com/gatsbyjs/gatsby-starter-kontent-homepage
+   ```
+
+1. **Create an empty Kontent project**
+
+1. **Run the Kontent setup script**
+
+   From your site's root directory, run:
+
+   ```sh
+   cd my-homepage
+   yarn setup
+   ```
+
+   This will run a script to populate your Kontent project with demo content.
```

Comment:
Perhaps a note here around where to find your project ID if you don't know would be good! I believe it's in your project space, under `/settings/api-keys`. E.g., https://app.kontent.ai/1da1fe7f-c2c1-00c6-b388-cfc83e665669/settings/api-keys

Reply:
I was thinking that it is enough that when you execute the script it will tell you where you can find it. But if you missed it there, it probably would be a good idea to put it there, because It can be easily missed in the script. What do you think, is it enough to have it in script?

Reply:
I'd say the effort involved in updating justifies having the instructions in both places — different users will use the software differently, and this way we can catch as many people as possible with the instructions!